src/transformers/modeling_tf_utils.py

```diff
@@ -35,6 +35,7 @@
 from huggingface_hub import Repository, list_repo_files
 from requests import HTTPError
 
+from . import DataCollatorWithPadding, DefaultDataCollator, PreTrainedTokenizer
```

Comment:
Not sure why there is the dependency on tokenization utils but if it's just for type hints, it should be protected by an `if TYPE_CHECKING`

Reply:
Done!